topics/sequence-analysis/tutorials/annotation-with-prokka/tutorial.md

```diff
@@ -0,0 +1,139 @@
+
+Prokka finds and annotates features (both protein coding regions and RNA genes, i.e. tRNA, rRNA) present on on a sequence. Note, Prokka uses a two-step process for the annotation of protein coding regions: first, protein coding regions on the genome are identified using [Prodigal](http://prodigal.ornl.gov/); second, the *function* of the encoded protein is predicted by similarity to proteins in one of many protein or protein domain databases. Prokka is a software tool that can be used to annotate bacterial, archaeal and viral genomes quickly, generating standard output files in GenBank, EMBL and gff formats. More information about Prokka can be found [here](https://github.com/tseemann/prokka).
+
+## Learning objectives
+{:.no_toc}
+
+At the end of this tutorial you should be able to:
+
+1. load a genome assembly into Prokka
+2. annotate the assembly using Prokka
+3. examine the annotated genome using JBrowse
+
+> ### Agenda
+>
+> In this tutorial, we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+## Import the data
+
+Prokka requires assembled contigs.
+
+> ### {% icon hands_on %} Hands-on: Obtaining our data
+>
+> 1. Make sure you have an empty analysis history. Give it a name.
+>
+>    > ### {% icon tip %} Starting a new history
+>    >
+>    > * Click the **gear icon** at the top of the history panel
+>    > * Select the option **Create New** from the menu
+>    {: .tip}
+>
+> 2. **Import Sample Data.** The data for this course may be available from a shared library in Galaxy
+> (ask your instructor). If this is not the case, you can upload it yourself.
+> - Option 1: From data library:
+>   - Navigate to the shared data library, you should find a file called `contigs.fasta`
+> - Option 2: From your computer:
+>   - Obtain data directly from Zenodo: [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1156405.svg)](https://doi.org/10.5281/zenodo.1156405)
+>   - Download `contigs.fasta`
+>   - Upload the file to your history.
+> <br><br>
+>
+{: .hands_on}
+
+## Annotate the genome
+
+Now we will run the tool called Prokka.
+
+> ### {% icon hands_on %} Hands-on: Annotate genome
+>
+> 1. **Prokka** {% icon tool %} with the following parameters (leave everything else unchanged)
+>    - contigs to annotate: `contigs.fasta`
+>    - Locus tag prefix (--locustag): P
+>    - Force GenBank/ENA/DDJB compliance (--compliant): No
+>    - Sequencing Centre ID (--centre): V
+>    - Genus Name: Staphylococcus  
+>    - Species Name: aureus  
+>    - Use genus-specific BLAST database: No  
+>    - Your tool interface should look like this:
+>    - ![prokka interface](images/interface.png)
+>    - Click Execute
+> <br><br>
+{: .hands_on}
+
+## Examine the output
+
+Once Prokka has finished, examine each of its output files.
+
+> ### {% icon hands_on %} Hands-on: output files
```

Comment:
I would not have put that as an hands-on: it looks more like a description and not really like something learners have to do. What do you think?

Reply:
Good idea - removed from hands-on